tests/e2e/backup_restore_suite_test.go

```diff
@@ -68,6 +68,8 @@ func prepareBackupAndRestore(brCase BackupRestoreCase, updateLastInstallTime fun
 		gomega.Eventually(lib.AreNodeAgentPodsRunning(kubernetesClientForSuiteRun, namespace), time.Minute*3, time.Second*5).Should(gomega.BeTrue())
 	}
 
+	// Velero does not change status of VSL objects. Users can only confirm if VSLs are correct configured when running a native snapshot backup/restore
```

Comment:
I think it was said in some scrum in 2021/2022 that we don't need VSL tests when implementing e2e due to "CSI is the future, we will not suggests VSL to anyone new onboarding". Just an info.

Reply:
your call. I guess keeping tests if functionality is there / not deprecated yet is fine..

Reply:
adding tests is fine yes. Pretty sure we had it at some point [here](https://github.com/openshift/oadp-operator/pull/568/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R8)